scipy/stats/_continuous_distns.py

```diff
@@ -8579,7 +8579,7 @@ class crystalball_gen(rv_continuous):
                             N A (B - x)^{-m}  &\text{for } x \le -\beta
                           \end{cases}
 
-    where :math:`A = (m / |\beta|)^n  \exp(-\beta^2 / 2)`,
+    where :math:`A = (m / |\beta|)^m  \exp(-\beta^2 / 2)`,
```

Comment:
May remove abs value from beta, which is positive. Or may leave until a some other time , too.

Reply:
I thought about that. The absolute values [seem to be standard](https://en.wikipedia.org/wiki/Crystal_Ball_function), so I wondered if maybe the existing code would work for negative beta, too, if we add the absolute values in the right places of the code.

Reply:
It does not. I think I'd like to leave that alone, as it's not incorrect - just not needed.